.vitepress/config.mts

```diff
@@ -35,6 +35,339 @@ function nav(): DefaultTheme.NavItem[] {
+          text: 'Trigger Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Advanced Filtering',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Block Stream',
+          collapsed: true,
+          items: [
+            {
+              text: 'Subscribe to Block Stream',
+            },
+            {
+              text: 'View output',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      text: 'Instructions',
+      collapsed: true,
+      items: [
+        {
+          text: 'Use Instructions',
+        },
+        {
+          text: 'Combine Instructions via Expressions',
+        },
+      ],
+    },
+    {
+      text: 'Metadata',
+      collapsed: true,
+      items: [
+        {
+          text: 'Set Key Value',
+        },
+        {
+          text: 'Remove Key Value',
+        },
+        {
+          text: 'Access Metadata',
+        },
+      ],
+    },
+    {
+      text: 'Peers',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register Peers',
+        },
+        {
+          text: "Check Current Peer's Load",
+        },
+        {
+          text: 'Find the Leader Among Running Peers',
+        },
+      ],
+    },
+    {
+      text: 'Queries',
+      collapsed: true,
+      items: [
+        {text: 'Use queries'},
+        {text: 'Filter query results'},
+        {text: 'Use Lazy Pagination'},
+        {text: 'Query Connected Peers'},
+      ],
+    },
+    {
+      text: 'Telemetry',
+      collapsed: true,
+      items: [
+        {
+          text: 'Check Status',
+        },
+        {
+          text: 'Get Metrics',
+        },
+        {
+          text: 'Monitor Iroha Performance',
+        },
+        {
+          text: 'Check Health',
+        },
+      ],
+    },
+    {
+      text: 'Transactions',
+      collapsed: true,
+      items: [
+        {
+          text: 'Create Transactions',
+        },
+        {
+          text: 'Submit Transactions',
+        },
+        {
+          text: 'Compile Transactions',
+        },
+        {
+          text: 'Use Multi-Signature Transactions',
+        },
+      ],
+    },
+    {
+      text: 'Triggers',
+      collapsed: true,
+      items: [
+        {
+          text: 'Data Triggers',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a Data Trigger',
+            },
+          ],
+        },
+        {
+          text: 'Time Triggers',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a Scheduled Trigger',
+            },
+            {
+              text: 'Register a Pre-commit Trigger',
+            },
+          ],
+        },
+        {
+          text: 'By-call Triggers',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a By-call Trigger',
+            },
+          ],
+        },
+      ],
+    },
```

Comment:
this part needs more details beside registering triggers. executing triggers? or just "work with {trigger type}"? I'm not sure

Reply:
there is nothing more to be added. Trigger is executed by the blockchain. You can add Unregister Trigger section (it's the same for all triggers)